backend/src/app.ts

```diff
@@ -16,6 +17,7 @@ app.use(cors());
 app.use(bodyParser.text({ type: '*/*' }));
 app.use('/mirror', mirror);
 app.use('/export/', exportRouter);
```

Comment:
All api endpoints should be under /api/ prefix to make reverse proxy configuration simpler

Reply:
Changed endpoints to be under /api